run_tests.py

```diff
@@ -46,16 +46,15 @@ def gdb_inspect(interactive):
                 '--nh', '--nx', 'mimiker.elf']
     gdb = pexpect.spawn(gdb_command, gdb_opts, timeout=3)
     gdb.expect_exact('(gdb)', timeout=5)
-    send_command(gdb, 'klog')
-    send_command(gdb, 'info registers')
-    send_command(gdb, 'backtrace')
-    send_command(gdb, 'list')
-    send_command(gdb, 'kthread')
+    for cmd in commands:
+        send_command(gdb, cmd)
+
     if interactive:
+        print('(gdb)', end='', flush=True)
```

Comment:
By default in interactive mode I'd print `backtrace` – that is always a starting point to understanding what went wrong.

But before that I'd like to see a listing of most useful commands related to debugging the kernel. I'd assume a user is familiar with _GNU gdb_, but not necessarily with our extensions to the command set like: `klog`, `cpu tlb`, `kthread`, `kdump` and so on...